agent/mission_solver/solver.py

```
@@ -0,0 +1,137 @@
+
+    filtered_quests = [quests[i] for i in useful_cols]
+    filtered_A = [[A[j][i] for i in useful_cols] for j in range(m)]
+    fn = len(filtered_quests)
+
+    logger.info(f"问题规模: {m} 个任务, {fn} 个候选副本 (从 {n} 个过滤)")
+
+    # 调用 HiGHS 求解
+    try:
+        import highspy
+    except ImportError:
+        logger.error("highspy 未安装，请运行: pip install highspy")
```

Ruff (0.15.10)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)